sockeye/inference.py

```diff
@@ -188,7 +188,15 @@ def sym_gen(source_seq_len: int):
 
             data_names = [C.SOURCE_NAME]
             label_names = []  # type: List[str]
-            return mx.sym.Group(decoder_init_states), data_names, label_names
+
+            # predict length ratios
+            predicted_length_ratios = []  # type: List[mx.nd.NDArray]
+            length_ratio_data_name = []  # type: List[str]
+            if hasattr(self, 'length_ratio'):
```

Comment:
why not `if self.length_ratio_task is not None`?

Reply:
FIxed

Reply:
fixed